lms/static/js/spec/main.js

```diff
@@ -703,6 +708,10 @@
         'js/spec/financial-assistance/financial_assistance_form_view_spec.js',
         'js/spec/groups/views/cohorts_spec.js',
         'js/spec/instructor_dashboard/certificates_bulk_exception_spec.js',
+        'js/spec/shoppingcart/shoppingcart_spec.js',
+        'js/spec/instructor_dashboard/instructor_dashboard_spec.js',
+        'js/spec/instructor_dashboard/ecommerce_spec.js',
+        'js/spec/instructor_dashboard/student_admin_spec.js',
```

Comment:
The only thing added here is the instructor dashboard spec. I'm not sure why the other three are showing up as added. I did have to rebase, but this should be comparing to master which should already have them?